src/pages/records/list.vue

```diff
@@ -46,6 +46,10 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.transactions-list {
+  overflow-y: scroll;
+  max-height: calc(100vh - 160px);
+}
```

Comment:
Why do we need `scroll` and max-height here? We already have a scroll in the `.records` selector.